weareinreach/inreach-catalog
# Feature/1201009454003445/thank you dialog

## Description

- Created a Thank You component.
- The thank you component is displayed at the end of each sign up flow
- When signing up as lawyer or provider, use can skip adding an organization by clicking the word 'skip'. The Thank you modal will appear after clicking skip.
- When signing up as lawyer or provider as user chooses to add an organization, the user will see a confirmation modal, when they click on Finish the Thank you modal will appear.

Also fixed an issue when the user clicked join organization before entering an organization value, there would be an error, added a validation measure. If no organization is specified a message is now displayed instructing the user to provide a valid organization.

## Asana ticket:
https://app.asana.com/0/1132189118126148/1201009454003445

## PR Checklist




## How to Test

1. Follow the standard signup test steps. 
2. The last modal should be a Thank you modal
3. This modal has two buttons. Find Resources, My Profile

Find Resources - goes to the search resources page
- click this and the thank you modal closes

logout and run thru another sign up flow, this time click on the My Profile button
My Profile - goes to account settings page of the logged in user
- click this and the thank you modal closes
